scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/synchronization/SynchronizationWrapper.java

```diff
@@ -0,0 +1,248 @@
+        return internalAPI.size(originator, taskId, channel);
+    }
+
+    @Override
+    public boolean isEmpty(String channel) throws InvalidChannelException {
+        return internalAPI.isEmpty(originator, taskId, channel);
+    }
+
+    @Override
+    public boolean containsKey(String channel, String key) throws InvalidChannelException {
+        return internalAPI.containsKey(originator, taskId, channel, key);
+    }
+
+    @Override
+    public boolean containsValue(String channel, Serializable value) throws InvalidChannelException {
+        return internalAPI.containsValue(originator, taskId, channel, value);
+    }
+
+    @Override
+    public <T extends Serializable> T get(String channel, String key) throws InvalidChannelException {
+        return internalAPI.get(originator, taskId, channel, key);
+    }
+
+    @Override
+    public <T extends Serializable> T put(String channel, String key, T value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.put(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public <T extends Serializable> T remove(String channel, String key) throws InvalidChannelException, IOException {
+        return internalAPI.remove(originator, taskId, channel, key);
+    }
+
+    @Override
+    public <T extends Serializable> void putAll(String channel, Map<? extends String, ? extends T> m)
+            throws InvalidChannelException, IOException {
+        internalAPI.putAll(originator, taskId, channel, m);
+    }
+
+    @Override
+    public void clear(String channel) throws InvalidChannelException, IOException {
+        internalAPI.clear(originator, taskId, channel);
+    }
+
+    @Override
+    public Set<String> keySet(String channel) throws InvalidChannelException {
+        return internalAPI.keySet(originator, taskId, channel);
+    }
+
+    @Override
+    public <T extends Serializable> Collection<T> values(String channel) throws InvalidChannelException {
+        return internalAPI.values(originator, taskId, channel);
+    }
+
+    @Override
+    public <T extends Serializable> Set<Map.Entry<String, T>> entrySet(String channel) throws InvalidChannelException {
+        return internalAPI.entrySet(originator, taskId, channel);
+    }
+
+    @Override
+    public <T extends Serializable> T getOrDefault(String channel, String key, T defaultValue)
+            throws InvalidChannelException {
+        return internalAPI.getOrDefault(originator, taskId, channel, key, defaultValue);
+    }
+
+    @Override
+    public <T extends Serializable> void forEach(String channel, String action)
+            throws InvalidChannelException, CompilationException {
+        internalAPI.forEach(originator, taskId, channel, action);
+    }
+
+    @Override
+    public <T extends Serializable> void replaceAll(String channel, String function)
+            throws InvalidChannelException, CompilationException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)